src/main/java/seedu/address/logic/Messages.java

```diff
@@ -36,12 +38,21 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
      */
     public static String format(Company company) {
         final StringBuilder builder = new StringBuilder();
-        builder.append(company.getName())
-                .append("; Phone: ")
-                .append(company.getPhone())
-                .append("; Email: ")
-                .append(company.getEmail())
-                .append("; Tags: ");
+        builder.append(company.getName());
+        if (!company.getPhone().value.equals(Phone.DEFAULT_NUMBER)) {
+            builder.append("; Phone: ")
+                    .append(company.getPhone());
+        }
+        builder.append("; Email: ")
+                .append(company.getEmail());
+        if (!company.getStartDate().getDate().equals(Date.getDefaultDate())
```

Comment:
The implementation of the Date class should not be exposed here. Handle equality check internally. In lieu of time, this change can be during the next PR.

Reply:
has been handled to use a method in Date to check if Date is present or not.